index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <style>
+      #navbar{
+        position: fixed
+        
+      }
+
+  @media (max-width: 600px) {
+  .facet_sidebar {
+    display: none;
+  }
+}
+    </style>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
The css/style.css file is linked, but there is no such file in the repository. Please add a CSS file for styling.